src/translations/en.json

```diff
@@ -1342,7 +1342,7 @@
           "description": "View the Home Assistant logs",
           "details": "Log Details ({level})",
           "search": "Search logs",
-          "no_issues_search": "No issues found for search term ''{term}'',
+          "no_issues_search": "No issues found for search term '{term}'",
```

Comment:
I thought `'` is the escape char, so `''` would be an escaped `'`. Have you tested this?

Reply:
Yup ;-)
![image](https://user-images.githubusercontent.com/2270505/151521858-6fe41724-9e61-422c-8769-6c9ac4d0a8f2.png)